spec/models/user_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe "validations" do
+    let(:user) { FactoryBot.create(:user) }
+    
+    it { should validate_presence_of(:name) }
+    it { should validate_presence_of(:email) }
+    it { should validate_presence_of(:password) }
+    it { should validate_presence_of(:phone) }
```

Comment:
Please write test so that it proves that your model is not accepting any string  for phone number. Thanks!

Reply:
provided a test. Thanks apu.